src/wallet/wallet.h

```diff
@@ -630,7 +630,7 @@ class CWalletTx
     // in place.
     std::set<uint256> GetConflicts() const NO_THREAD_SAFETY_ANALYSIS;
 
-    void SetConf(Status status, const uint256& block_hash, int posInBlock);
+    void SetConf(Status status, const uint256& block_hash, int posInBlock, int block_height);
```

Comment:
In commit "Add block_height field in struct Confirmation" (de8fe7d717042823f2aa429be2a260904a10b42e)

Could you move the block_height argument after block_hash to be consistent with `SyncTransaction` argument order and `Confirmation` struct order?